csharp/Facebook.Yoga/mac/ApiDefinition.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+using AppKit;
+using Foundation;
+using ObjCRuntime;
+using CoreGraphics;
+
+namespace Facebook.Yoga.Mac
+{
+	// We're actually abusing the Xamarin.Mac binding project
```

Comment:
Apparently we need empty ApiDefinition.cs but don't bind static lib. Can you update the comments?